foundryContract/src/libraries/MajorityVoting.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MIT
```

Comment:
I don't think we need this file. It is used for Request-Response. Please remove it.